ddtrace/appsec/iast/_patch.py

```diff
@@ -1,11 +1,15 @@
 import ctypes
 import gc
+from typing import Any
```

Comment:
add TYPE_CHECKING:


```python
if TYPE_CHECKING:  # pragma: no cover
    from typing import Any
    from typing import Dict
```

Reply:
IIUC that makes sense [if you're importing from an expensive module](https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING), but I don't see the point here as we're importing TYPE_CHECKING from typing as well.